packages/strapi-plugin-content-type-builder/admin/src/containers/RelationForm/RelationBox.js

```diff
@@ -53,11 +53,13 @@ const RelationBox = ({
             didCheckErrors={didCheckErrors}
             errors={errors}
             label="Field Name"
-            disabled={value === '-' || nature === 'oneWay'}
+            disabled={
+              value === '-' || nature === 'oneWay' || nature === 'manyWay'
+            }
             name={main ? 'name' : 'key'}
             onChange={onChange}
             type="text"
-            value={nature === 'oneWay' ? '-' : value}
+            value={nature === 'oneWay' || nature === 'manyWay' ? '-' : value}
```

Comment:
It seems like you have two RelationBox components? Can't you use the same one?

Reply:
Will use the same when RelationFormGroup and RelationForm are the same

Reply:
Seems to me they are pretty much the same. We will use the same one in another PR.